doc/build-windows.md

```diff
@@ -117,6 +117,20 @@ Once the source code is ready the build steps are below.
     CONFIG_SITE=$PWD/depends/x86_64-w64-mingw32/share/config.site ./configure --prefix=/
     make
 
+Ubuntu Bionic Beaver 18.04:
+
+    sudo update-alternatives --config x86_64-w64-mingw32-g++ # Set the default mingw32 g++ compiler option to posix.
+
+Bionic Beaver has a modern mingw version. You can therefore use it to build
+Bitcoin Qt with Qt v5.9.4. Follow these steps:
+
+    cd depends
+    make HOST=x86_64-w64-mingw32 QT_59=1
+    cd ..
+    ./autogen.sh # not required when building from tarball
+    CONFIG_SITE=$PWD/depends/x86_64-w64-mingw32/share/config.site ./configure --prefix=/ --enable-qt59
+    make
+
```

Comment:
This whole section is added in #13246, so feel free to remove this to avoid merge conflicts.